docs/modules/ROOT/pages/ws-security.adoc

```diff
@@ -3,6 +3,19 @@
 
 The CXF framework's https://cxf.apache.org/docs/ws-security.html[WS-Security] implementation is based on https://ws.apache.org/wss4j/user_guide.html[WSS4J]. The example below shows how to integrate with WSS4J using interceptors.
 
+To use WSS4J interceptors in your code, firstly you should add CXF's WSS4J dependency (Replace `version` with CXF version used by Quarkus).
```

Comment:
- You are adding CXF's **WS-Security** dependency _(not WSS4J)_
- I don't believe we need to say _"(Replace `version` with CXF version used by Quarkus)"_ .... for example, if someone is using the CXF BOM, then they don't need a version specified here at all.  Would suggest we remove this statement, along with the version tag in the XML block.  A basic understanding of Maven can be expected of the extension users, so I think we can omit this level of detail.

Reply:
Thanks for suggestions, I changed version statement like "_you should add latest version_"  to satisfy any curiosities,  if it needs better statement, please let me know.